spec/recipes/install_forwarder_spec.rb

```diff
@@ -2,87 +2,35 @@
 
 describe 'chef-splunk::install_forwarder' do
   context 'debian family' do
+    let(:url) { 'http://splunk.example.com/forwarder/package.deb' }
+
     let(:chef_run) do
       ChefSpec::ServerRunner.new(
-        step_into: ['splunk_installer'],
         platform: 'ubuntu',
         version: '16.04'
       ) do |node|
-        node.normal['splunk']['forwarder']['url'] = 'http://splunk.example.com/forwarder/package.deb'
+        node.force_default['splunk']['forwarder']['url'] = url
       end.converge(described_recipe)
     end
 
-    it 'caches the package with remote_file' do
-      expect(chef_run).to create_remote_file_if_missing("#{Chef::Config[:file_cache_path]}/package.deb")
-    end
-
-    it 'installs the package with the downloaded file' do
-      expect(chef_run).to install_dpkg_package('splunkforwarder').with(
-        'source' => "#{Chef::Config[:file_cache_path]}/package.deb"
-      )
+    it 'ran the splunk installer' do
+      expect(chef_run).to run_splunk_installer('splunkforwarder').with(url: url)
     end
   end
 
   context 'redhat family' do
+    let(:url) { 'http://splunk.example.com/forwarder/package.rpm' }
     let(:chef_run) do
       ChefSpec::ServerRunner.new(
-        step_into: ['splunk_installer'],
         platform: 'centos',
-        version: '6.9'
-      ) do |node|
-        node.normal['splunk']['forwarder']['url'] = 'http://splunk.example.com/forwarder/package.rpm'
-      end.converge(described_recipe)
-    end
-
-    it 'caches the package with remote_file' do
-      expect(chef_run).to create_remote_file_if_missing("#{Chef::Config[:file_cache_path]}/package.rpm")
-    end
-
-    it 'installs the package with the downloaded file' do
-      expect(chef_run).to install_rpm_package('splunkforwarder').with(
-        'source' => "#{Chef::Config[:file_cache_path]}/package.rpm"
-      )
-    end
-  end
-
-  context 'omnios family' do
-    let(:chef_run) do
-      ChefSpec::ServerRunner.new(
-        step_into: ['splunk_installer'],
-        platform: 'omnios',
-        version: '151018'
+        version: '7.6'
```

Comment:
if you set this to jus '7' then chefspec will use the latest 7.x and you'll avoid deprecation warnings down the line when we eventually deprecated 7.6 dumps